### DISCUSSION TITLE
some sort of fix for the gif issue or a bypass sprite upload 10mb limit

### DISCUSSION THREAD

**Discussion by thesoaplord** [ScratchAddons/ScratchAddons#3981] [Ideas]
if you don't know about the "gif issue" i'm referring to, this: https://scratch.mit.edu/discuss/topic/267340/?page=1#post-2689199 should help you get an idea. i know about that importing the gif as a sprite actually treats it like a gif, however splitting larger animations into tons of short gifs as separate sprites to bypass this would be quite time-consuming(and in some cases difficult.)

if neither of these are possible it's okay, and i apologize for looking stupid.

**Comment by WorldLanguages** [comment]
Are you a Windows user? I think file explorer is the one to blame for sending the files in wrong order. Can you screenshot the file explorer when trying to upload many images at once?

**Reply by thesoaplord** [reply]
i'm not exactly sure what you meant, but here you go. and yes, i am a windows user
![screenshot](https://user-images.githubusercontent.com/96513037/147133934-9b4c0982-1f67-4f16-ad8c-963c8b3cdd83.PNG)

**Reply by apple502j** [reply]
See https://github.com/LLK/scratch-gui/issues/5138
